README.md

```diff
@@ -13,101 +13,103 @@ See [Contributing](.github/CONTRIBUTING.md).
 
 ## Table of contents
```

Comment:
You have changed the table of contents (probably by mistake) please remove these changes.